tests/test_handle.py

```diff
@@ -33,3 +33,14 @@ def test_invalid_model_handle(self) -> None:
     def test_invalid_version_model_handle(self) -> None:
         with self.assertRaises(ValueError):
             parse_model_handle("google/bert/tensorFlow2/answer-equivalence-bem/invalid-version-number")
+
+    def test_dataset_handle(self) -> None:
```

Comment:
Add tests for versioned and unversioned dataset handles and name the tests accordingly.